ui/src/components/visit-us/VisitUsPage.js

```diff
@@ -0,0 +1,192 @@
+                Tong
+                <br />
+                <Link
+                  href="https://bit.ly/directions-organize"
+                  color="#0E66CC"
+                  isExternal
+                >
+                  <Text as="u">bit.ly/directions-organize</Text>
+                </Link>
+              </Text>
+              <Spacer />
+              <Center w="100%" px={1}>
+                <LinkBox
+                  style={{ backgroundOrigin: "border-box" }}
+                  as="button"
+                  transition="all 0.2s cubic-bezier(.08,.52,.52,1)"
+                  boxShadow="2px 1000px 1px #fff inset"
+                  boxSizing="border-box"
+                  border="solid 3px transparent"
+                  borderRadius={10}
+                  backgroundImage="linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0)), linear-gradient(to right, #00B5D8, #F6AD55)"
+                  backgroundClip="content-box, border-box"
+                  py={3}
+                  w="100%"
+                  fontSize={["md", "lg"]}
+                  _hover={{ boxShadow: "none", color: "#fff" }}
+                >
+                  <LinkOverlay href="https://bit.ly/hmcc-sc-registration">
+                    Sign Up for In-person Sunday Celebration
+                  </LinkOverlay>
+                </LinkBox>
+              </Center>
+              <Text textAlign="center" fontSize="sm">
+                *Pre-registration opens every Monday at 8PM
+              </Text>
+            </VStack>
+            <AspectRatio flex={5.75} ratio={16 / 9}>
+              <iframe
+                title="Organize Kwun Tong - Google Maps"
+                src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3691.2102241954112!2d114.22135071553005!3d22.307887848255323!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3404014806001b5d%3A0x895969fc5cf86457!2sOrganize%20Kwun%20Tong!5e0!3m2!1sen!2shk!4v1628957852778!5m2!1sen!2shk"
+                loading="lazy"
+              ></iframe>
+            </AspectRatio>
+          </Flex>
+        </Box>
+        <Box
+          borderWidth="1px"
+          borderRadius="20"
+          bgImage={`url('${process.env.PUBLIC_URL}/images/visitus/join-us-online.png')`}
+          bgPosition="center"
+          bgSize="cover"
+          px={6}
+          pt={[4, 6]}
+          pb={4}
+        >
+          <VStack align="stretch" spacing={12}>
+            <Heading
+              as="h3"
+              fontSize={["2xl", "4xl"]}
+              fontWeight={[400, 700]}
+              color="white"
+              maxW={650}
+              textAlign={["center", "initial"]}
+            >
+              If you are not able to join in-person, you could also...
+            </Heading>
+            <Heading
+              as="h3"
+              fontSize={["2xl", "4xl"]}
+              color="white"
+              textAlign={["center", "right"]}
+            >
+              <Link href="/online">
```

Comment:
`target="_blank"` and also i think this will still link to hongkong.hmcc.net/online or whatever domain we choose for our old website